webpack.config.js

```diff
@@ -15,18 +16,22 @@ const PATHS = {
 
 const plugins = [];
 // don't include momentjs locales (large)
-plugins.push(new webpack.IgnorePlugin(/^\.\/locale$/, [/moment$/]));
+plugins.push(new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/));
```

Comment:
This may have a conflict with the moment js update.

Reply:
Great point. But it is a river I will cross when I get to that river (aka when fixing an upcoming merge-conflict battle with dev). Are you okay with this approach?